test/Datadog.Trace.ClrProfiler.IntegrationTests/Helpers/WebServerSpanExpectation.cs

```diff
@@ -1,104 +1,24 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Datadog.Trace;
-using Datadog.Trace.TestHelpers;
-
 namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
-    public class WebServerSpanExpectation
+    public class WebServerSpanExpectation : SpanExpectation
     {
-        public string OriginalUri { get; set; }
-
-        public string OperationName { get; set; }
-
-        public string ServiceName { get; set; }
-
-        public string Type { get; set; }
-
-        public string ResourceName { get; set; }
-
-        public string StatusCode { get; set; }
-
-        public string HttpMethod { get; set; }
-
-        public Func<MockTracerAgent.Span, List<string>> CustomAssertion { get; set; }
-
-        public virtual bool IsSimpleMatch(MockTracerAgent.Span span)
+        public WebServerSpanExpectation(string serviceName, string operationName)
+            : base(serviceName, operationName, SpanTypes.Web)
```

Comment:
Are you trying to call the other constructor from this one? If so, I think you'll need to change `base` to `this` (see https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-constructors). At the moment I'm worried that if this one is called that the `RegisterTagExpectation` calls will not be added

Reply:
Good catch, thanks